tensorflow/lite/experimental/examples/unity/TensorFlowLitePlugin/Assets/TensorFlowLite/SDK/Scripts/Interpreter.cs

```diff
@@ -89,18 +94,95 @@ public class Interpreter : IDisposable
           tensor, tensorDataPtr, Buffer.ByteLength(outputTensorData)));
     }
 
+    public string GetInputTensorInfo(int index) {
```

Comment:
What about having a Tensor struct that has the tensor name, shape, type and optional quantization params?

Reply:
Fixed it

Reply:
That's nice. Updated to returning the TensorInfo struct. (since `Tensor` is a little confusing name with     TfLiteTensor).